### ISSUE TITLE
Calling `eth_getChainConfigs` doesn't contain an `upgrades` section anymore

### ISSUE THREAD

**Issue by holisticode** [ava-labs/subnet-evm#543]
**Describe the bug**
In response to the following code:
```
const chainConfigAPI = "eth_getChainConfig"
if err := rpcClient.CallContext(ctx, &chainConfig, chainConfigAPI); err != nil {
		return err
	}

fmt.Println(string(chainConfig))
```

I am getting
```{
    "byzantiumBlock": 0,
    "chainId": 99999,
    "constantinopleBlock": 0,
    "eip150Block": 0,
    "eip150Hash": "0x2086799aeebeae135c246c65021c82b4e15a2c451340993aacfd2751886514f0",
    "eip155Block": 0,
    "eip158Block": 0,
    "feeConfig": {
        "gasLimit": 8000000,
        "targetBlockRate": 2,
        "minBaseFee": 25000000000,
        "targetGas": 15000000,
        "baseFeeChangeDenominator": 36,
        "minBlockGasCost": 0,
        "maxBlockGasCost": 1000000,
        "blockGasCostStep": 200000
    },
    "homesteadBlock": 0,
    "istanbulBlock": 0,
    "muirGlacierBlock": 0,
    "petersburgBlock": 0,
    "subnetEVMTimestamp": 0
}
```

I am missing an `upgrades` section in this response as I was getting until 0.4.9. Our test in `avalanche-cli` was using this to evaluate if the upgrade was successful.

**To Reproduce**
Run the above code on a node calling the `eth_getChainConfig"`` API on a node running 0.4.10

**Expected behavior**
Expected an `upgrades` section in the JSON

**Operating System**
linux amd64

**Comment by aaronbuchwald** [comment]
Thanks for putting this up 🙌

**Comment by ceyonur** [comment]
This is shipped with the v0.4.10, feel free to test and close this @holisticode

**Comment by holisticode** [comment]
As of my testing, this seems to be fixed. Thank you!.
